baseline_testing/testcase_bestest_air_scenario.py

```diff
@@ -0,0 +1,131 @@
+# -*- coding: utf-8 -*-
+"""
+This script demonstrates baseline testing of test case called "bestest_air".
+
+"""
+
+# GENERAL PACKAGE IMPORT
+# ----------------------
+import sys
+import os
+sys.path.insert(0, '/'.join(os.path.dirname(os.path.abspath(__file__))))
+from examples.python.interface import control_test
+import pandas as pd
+import json
+import itertools
+import datetime as dt
+import time
+
+
+def run(modelname, scenario, start_time = 0, warmup_period = 0, length = 48*3600,  plot=False):
+    """Run test case.
+    Parameters
+    ----------
+    modelname: str
+        Name of the testcase.
+        
+    scenario: dict, optional
+        Dictionary defining the predefined test scenario.
+        {'time_period': str, 'electricity_price': str}.
+        If specified, start_time, warmup_period, and length not used.
+        
+    start_time: int, optional
+        Simulation start time in seconds from midnight January 1st.
+        Not used if scenario defined.
+        
+    warmup_period: int, optional
+        Simulation warmup-period in seconds before start_time.
+        Not used if scenario defined.
+        
+    length: int, optional
+        Simulation duration in seconds (e.g., 24*3600 is a 1 day simulation).
+        Not used if scenario defined.
+
+    plot : bool, optional
+        True to plot timeseries results.
+        Default is False.
+
+    Returns
+    -------
+    kpi : dict
+        Dictionary of core KPI names and values.
+        {kpi_name : value}
+    res : dict
+        Dictionary of trajectories of inputs and outputs.
+    custom_kpi_result: dict
+        Dictionary of tracked custom KPI calculations.
+        Empty if no customized KPI calculations defined.
+
+    """
+
+    # RUN THE CONTROL TEST
+    # --------------------
+    control_module = 'examples.python.controllers.{}'.format(modelname)
```

Comment:
I suggest creating one controller that simply sets ``u={}`` instead of having a controller for each test case that sets ``_activate=0`` for every variable anyway.  Writing an empty dictionary to the ``/advance`` call in BOPTEST will simply run the baseline control.  I suggest calling this controller module``baseline.py``, thereby allowing for removing all the test case -specific controller modules.

Reply:
The baseline.py module file is utilized for executing the baseline control for each test case. It simplifies the process by setting u={}.